### PR TITLE
Add Thirdweb Page

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds Thirdweb content

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+import { readContract } from 'thirdweb';
+
+const number = await readContract({
+  contract: contract,
+  method: 'number',
+  params: [],
+});
+```
+
+Did you know? With the [thirdweb CLI](https://portal.thirdweb.com/cli){target=\_blank}, you can easily and generate functions for all of the possible calls to a contract. To do so, run the following command in the command line:
```
Comment (augustocollado) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:285]:
```suggestion
Did you know? With the [thirdweb CLI](https://portal.thirdweb.com/cli){target=\_blank}, you can easily generate functions for all of the possible calls to a contract. To do so, run the following command in the command line: 
```

**Review by augustocollado — CHANGES_REQUESTED** [review]
Besides the minor grammar error, it looks great!
I noticed that the thirdweb word is always written in lowercase, even in the menu or when it's the first word of the sentence (like in the intro or the connect button section). Is that intended?

**Comment by themacexpert** [comment]
> Besides the minor grammar error, it looks great! I noticed that the thirdweb word is always written in lowercase, even in the menu or when it's the first word of the sentence (like in the intro or the connect button section). Is that intended?

Yup indeed! haha I think it's part of their branding or something

**Review by augustocollado — DISMISSED** [review]
LGTM 🚀

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+The below code snippet demonstrates how to initialize and connect a MetaMask wallet using the thirdweb SDK, then sign and send a transaction, retrieving the transaction hash. This process is applicable to any of the 300+ wallet connectors supported by the SDK.
+
+???+ code "initialize.ts"
+    ```typescript
+    --8<-- 'code/builders/toolkit/ethereum-api/dev-env/thirdweb/initialize.ts'
+    ```
+
+### Get Contract {: #get-contract }
+
+To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/references/typescript/v5/getContract){target=\_blank} method. As an example, you could fetch data from an [incrementer contract on the Tanssi Demo EVM Appchain](https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/address/0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D?tab=contract){target=\_blank}.
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:194]:
```suggestion
To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/references/typescript/v5/getContract){target=\_blank} method. As an example, you could fetch data from an [incrementer contract on the Tanssi demo EVM appchain](https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/address/0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D?tab=contract){target=\_blank}.
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+
+```typescript
+import { getContract } from 'thirdweb';
+import { client } from './client';
+
+const myContract = getContract({
+  client,
+  chain: tanssi,
+  address: 0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D, // Incrementer contract address on Demo EVM Appchain
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:203]:
```suggestion
  address: 0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D, // Incrementer contract address on demo EVM appchain
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+```typescript
+import { prepareTransaction, toWei } from 'thirdweb';
+
+const transaction = prepareTransaction({
+  // The account that will be the receiver
+  to: '0x456...',
+  // The value is the amount of ether you want to send with the transaction
+  value: toWei('1'),
+  // The chain to execute the transaction on. This assumes you already set up
+  // the Tanssi Demo EVM Appchain as a custom chain as shown in the configure chain section
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:252]:
```suggestion
  // the Tanssi demo EVM appchain as a custom chain, as shown in the configure chain section
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 5]
```diff
@@ -0,0 +1,425 @@
...
+});
+```
+
+Did you know? With the [thirdweb CLI](https://portal.thirdweb.com/cli){target=\_blank}, you can easily generate functions for all of the possible calls to a contract. To do so, run the following command in the command line: 
+
+```bash
+npx thirdweb generate INSERT_CHAIN_ID/INSERT_CONTRACT_ADDRESS
+```
+
+Both the chain ID and the contract address are required. As an example, if you wanted to generate the functions for the [incrementer contract on the Tanssi Demo EVM Appchain](https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/address/0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D?tab=contract){target=\_blank} , you would use the following command:
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:291]:
```suggestion
Both the chain ID and the contract address are required. As an example, if you wanted to generate the functions for the [incrementer contract on the Tanssi demo EVM appchain](https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/address/0xC12f6fA2d1CA8f875bD25555e8883f1dDa40a93D?tab=contract){target=\_blank} , you would use the following command:
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 6]
```diff
@@ -0,0 +1,425 @@
...
+    - Uploading your contract source code (ABI) to IPFS
+
+2. When it is completed, it will open a dashboard interface to finish filling out the parameters
+
+    - `_name` - contract name
+    - `_symbol` - symbol or "ticker"
+    - `_royaltyRecipient` - wallet address to receive royalties from secondary sales
+    - `_royaltyBps` - basis points (bps) that will be given to the royalty recipient for each secondary sale, e.g. 500 = 5%
+
+3. Select the desired network, e.g., the Tanssi Demo EVM Appchain or your own Appchain
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:95]:
```suggestion
3. Select the desired network, e.g., the Tanssi demo EVM appchain or your own appchain
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 7]
```diff
@@ -0,0 +1,425 @@
...
+
+# Using thirdweb on Tanssi
+
+## Introduction {: #introduction }
+
+[thirdweb](https://thirdweb.com){target=\_blank} is a complete Web3 development framework that provides everything you need to develop smart contracts, build dApps, and more.
+
+With thirdweb, you can access tools to help you through every phase of the dApp development cycle. You can create your own custom smart contracts or use any of thirdweb's prebuilt contracts to get started quickly. From there, you can use thirdweb's CLI to deploy your smart contracts. Then you can interact with your smart contracts by creating a Web3 application using the language of your choice, including but not limited to React and TypeScript. 
+
+This guide will show you some of the thirdweb features you can use to develop smart contracts and dApps on Tanssi EVM Appchains. To check out all of the features thirdweb has to offer, please refer to the [thirdweb documentation site](https://portal.thirdweb.com){target=\_blank}.
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:14]:
```suggestion
This guide will show you some of the thirdweb features you can use to develop smart contracts and dApps on Tanssi EVM appchains. To check out all of the features thirdweb has to offer, please refer to the [thirdweb documentation site](https://portal.thirdweb.com){target=\_blank}. 
```

File: builders/toolkit/ethereum-api/dev-env/thirdweb.md [hunk 8]
```diff
@@ -0,0 +1,425 @@
...
+    }
+    ```
+
+    This contract inherits the functionality of `ERC721Base` through the following steps:
+
+    - Importing the `ERC721Base` contract
+    - Inheriting the contract by declaring that your contract is an `ERC721Base` contract
+    - Implementing any required methods, such as the constructor
+
+5. After modifying your contract with your desired custom logic, you can deploy it to a Tanssi EVM Appchain using [Deploy](https://portal.thirdweb.com/contracts/deploy/overview){target=\_blank}. That will be covered in the next section!
```
Comment (eshaben) [inline builders/toolkit/ethereum-api/dev-env/thirdweb.md:59]:
```suggestion
5. After modifying your contract with your desired custom logic, you can deploy it to a Tanssi EVM appchain using [Deploy](https://portal.thirdweb.com/contracts/deploy/overview){target=\_blank}. That will be covered in the next section!
```

**Review by eshaben — COMMENTED** [review]
Just noticed some minor stuff

**Review by augustocollado — APPROVED** [review]

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**